src/schema.py

```diff
@@ -95,15 +99,104 @@ class Meta:
         model = CapacityModel
 
 
+# MARK: - User
+
+
+class User(SQLAlchemyObjectType):
+    class Meta:
+        model = UserModel
+
+
+class UserInput(graphene.InputObjectType):
+    net_id = graphene.String(required=True)
+    giveaway_id = graphene.Int(required=True)
+
+
+# MARK: - Giveaway
+
+
+class Giveaway(SQLAlchemyObjectType):
+    class Meta:
+        model = GiveawayModel
+
+    user_ids = graphene.List(lambda: User)
+
+    def resolve_user_ids(self, info):
+        query = User.get_query(info=info).filter(UserModel.giveaway_id == self.id)
+        return query
+
+
+# MARK: - Giveaway
+
+
+class GiveawayInstance(SQLAlchemyObjectType):
+    class Meta:
+        model = GiveawayInstanceModel
+
+
 # MARK: - Query
 
 
 class Query(graphene.ObjectType):
     gyms = graphene.List(Gym)
+    users_by_giveawayid = graphene.List(User, id=graphene.Int())
 
     def resolve_gyms(self, info):
         query = Gym.get_query(info)
         return query.all()
 
 
-schema = graphene.Schema(query=Query)
+def resolve_users_by_giveaway_id(self, info, id):
+    query = User.get_query(info).filter(UserModel.giveaway_ids.any(GiveawayInstance.giveaway_id == id))
+    return query.all()
```

Comment:
```suggestion
    def resolve_get_users_by_giveaway_id(self, info, id):
        entries = GiveawayInstance.get_query(info).filter(GiveawayInstanceModel.giveaway_id == id).all()
        users = [User.get_query(info).filter(UserModel.id == entry.user_id).first() for entry in entries]
        return users
```
This should be indented. You can also change the way you're filtering to make it more time efficient.